vault/resource_mount.go

```diff
@@ -511,3 +517,16 @@ func mountOptions(d *schema.ResourceData) map[string]string {
 	}
 	return options
 }
+
+func tuneMountWithMap(c *api.Client, ctx context.Context, path string, config map[string]interface{}) error {
```

Comment:
nit: typically the context.Context is the first argument, would you mind updating this?